From 57f04eaa7a1fe0a1b84815bc1b4d46f9a9019443 Mon Sep 17 00:00:00 2001
From: Tomasz Figa <t.figa@samsung.com>
Date: Tue, 19 Mar 2013 15:24:14 +0100
Subject: [PATCH 0099/1302] ARM: dts: exynos4x12: Add cpufreq node

This patch adds cpufreq node with SoC specific paramaters for
cpufreq-exynos driver.

Signed-off-by: Tomasz Figa <t.figa@samsung.com>
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 arch/arm/boot/dts/exynos4x12.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/exynos4x12.dtsi b/arch/arm/boot/dts/exynos4x12.dtsi
index 8d0e3d83..c0edb05 100644
--- a/arch/arm/boot/dts/exynos4x12.dtsi
+++ b/arch/arm/boot/dts/exynos4x12.dtsi
@@ -261,4 +261,12 @@
 			};
 		};
 	};
+
+	cpufreq {
+		compatible = "samsung,exynos-cpufreq";
+		clocks = <&clock 12>, <&clock 19>, <&clock 9>, <&clock 20>;
+		clock-names = "arm_clk", "mout_core", "mout_mpll_user_c",
+								"mout_apll";
+		status = "disabled";
+	};
 };
-- 
1.8.3.2

